_opensearch/rest-api/cat/cat-cluster_manager.md

```diff
@@ -7,7 +7,7 @@ nav_order: 30
 has_children: false
 ---
 
-# CAT cluster_manager
+# cat cluster_manager
```

Comment:
should we keep the CAT acronym in uppercase? 
then we could also update the other CAT API titles under this index to keep it consistent: 
https://opensearch.org/docs/latest/opensearch/rest-api/cat/index/